recipes/jamo/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
Please switch to the GitHub source tar so the LICENSE is extracted from the tar instead of being manually added here.

Reply:
Fixed

Reply:
The Github branch is incorrectly tagged. The latest version is `0.4.1`, but the global version variable of the corresponding Github tag says `0.4.2` (https://github.com/JDongian/python-jamo/blob/v0.4.1/jamo/__init__.py#L16). I think that using the PyPi distribution and copying the license file will cause fewest problems.

Reply:
Why would the PyPI distribution not have the same problem? In both cases, the tag/release is 0.4.1, but the __init__ claims it is 0.4.2. If the mismatch is a problem, then skip the 0.4.1 release and raise an issue upstream.